packages/babel-types/test/fixtures/identifier.js

```diff
@@ -0,0 +1,16 @@
+// spec.Identifier (e.g. BindingIdentifier).
+// value => valid (ES2015 strict).
+module.exports = [
+  ["x", true],
+
+  // IdentifierName, ReservedWord :: Keyword.
+  ["default", false],
+
+  // ReservedWord's
+  ["null", false],
+  ["true", false],
+  ["false", false],
+
+  // Not a lexical IdentifierName.
+  ["bound x", false]
+];
```

Comment:
Ok thanks. I wasn't trying to be exhaustive -- I figured this is good enough to start with. People can add to it if they like.